### ISSUE TITLE
Zenodo for citing rrtm?

### ISSUE THREAD

**Issue by serbinsh** [ashiklom/rrtm#8]
@ashiklom hey, do you have any plans to create a Zenodo project for this package?  Asking as I would like to cite the code and if its in Zenodo then i can use their DOI.  

If not we can just include the Github URL.  Thanks!

**Comment by ashiklom** [comment]
Yes, great idea. Remind me on Slack early next week -- I'm currently on
travel.
